bigframes/ml/compose.py

```diff
@@ -79,13 +105,25 @@ def _keys(self):
     def transformers_(
         self,
     ) -> List[
-        Tuple[str, Union[preprocessing.PreprocessingType, impute.SimpleImputer], str]
+        Tuple[
+            str,
+            Union[
+                preprocessing.PreprocessingType,
+                impute.SimpleImputer,
+                SQLScalarColumnTransformer,
+            ],
+            str,
+        ]
     ]:
         """The collection of transformers as tuples of (name, transformer, column)."""
         result: List[
             Tuple[
                 str,
-                Union[preprocessing.PreprocessingType, impute.SimpleImputer],
+                Union[
+                    preprocessing.PreprocessingType,
+                    impute.SimpleImputer,
+                    SQLScalarColumnTransformer,
+                ],
```

Comment:
We're getting enough of these now that it's probably worth creating a constant to represent this union of types.

See: https://github.com/googleapis/python-bigquery-dataframes/blob/6b34244e9ebe747161da21af2f31ee537b653093/bigframes/dtypes.py#L31-L38 for an example.